src/opm/parser/eclipse/EclipseState/Grid/SatfuncPropertyInitializers.cpp

```diff
@@ -85,7 +85,7 @@ namespace {
         const auto gas    = ph.active(::Opm::Phase::GAS);
 
         const auto threeP = gas && oil && wat;
-        const auto twoP = (!gas && oil && wat) || (gas && oil && !wat) ;
+        const auto twoP = (!gas && oil && wat) || (gas && oil && !wat) || (gas && !oil && wat) ;
```

Comment:
Now that we recognise all two-phase combinations we **might** alternatively wish to express this as
```C++
const auto twoP = (gas + oil + wat) == 2;
```
but it's not clear to me that that's easier to read.

Reply:
I like the suggestion and I think it is easier to read . Then also use:  const auto threeP = (gas + oil + wat) == 3; ?